.lycheeexclude

```diff
@@ -0,0 +1,166 @@
+# Local or predefined end points
```

Comment:
do we need to add a license at the top of this file.

Reply:
Not sure about the SOP for licenses. Do you mean including lychee license or creating a license comment for `.lycheeexclude` file?  The file is created by us. It is just a concatenated string for all the links we want to exclude. The tool currently only support `--exclude` with particular link but we have a lot of links to exclude, so we create a list separately and load it to the command.

Reply:
> creating a license comment for .lycheeexclude file? The file is created by us

This one related to this: https://github.com/opensearch-project/.github/issues/21

Reply:
Thanks, will add